packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -32,197 +40,151 @@ Chains not supported yet due fireblocks or zerodev doesn't supported
     SOLANA
     CARDANO
 */
-export const EVMBlockchain = {
-    ETHEREUM: "ethereum",
-    POLYGON: "polygon",
-    BSC: "bsc",
-    OPTIMISM: "optimism",
-    ARBITRUM: "arbitrum",
-} as const;
-export type EVMBlockchain = (typeof EVMBlockchain)[keyof typeof EVMBlockchain];
-
-export const BlockchainTestNet = {
-    GOERLI: "goerli",
-    SEPOLIA: "sepolia",
-    MUMBAI: "mumbai",
-    ZKATANA: "zkatana",
-} as const;
-export type BlockchainTestNet = (typeof BlockchainTestNet)[keyof typeof BlockchainTestNet];
-
-export const EVMBlockchainWithTestnet = {
-    ...EVMBlockchain,
-    ...BlockchainTestNet,
-} as const;
-
-export type EVMBlockchainWithTestnet = (typeof EVMBlockchainWithTestnet)[keyof typeof EVMBlockchainWithTestnet];
-
-export const Blockchain = {
-    ...EVMBlockchain,
-    ...BlockchainTestNet,
-} as const;
-export type Blockchain = (typeof Blockchain)[keyof typeof Blockchain];
-
-export function getAssetIdByBlockchain(chain: Blockchain) {
+
+export function getAssetIdByBlockchain(chain: BlockchainIncludingTestnet) {
```

Comment:
this is specifically for fireblocks, right?
What do you think about renaming the function to denote that? Like `getFireblocksAssetId`?